### PR TITLE
added snapchat icon into ui-kit

### LINKED ISSUE #93: add snapchat icon

### PR DESCRIPTION
**Pull request by manojkannekanti**
resolves #93

### CODE REVIEW

File: src/icons/snapchat.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+export default ({ width, height }) => {
+  return (
+    <svg
+      id='Bold'
```
Comment (NejcZdovc) [inline src/icons/snapchat.jsx:4]: please remove

File: src/icons/snapchat.jsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+      enableBackground='new 0 0 24 24'
```
Comment (NejcZdovc) [inline src/icons/snapchat.jsx:5]: is this needed?

File: src/icons/snapchat.jsx (hunk 1 above)
Comment (manojkannekanti) [reply src/icons/snapchat.jsx]: removed

File: src/icons/snapchat.jsx (hunk 2 above)
Comment (manojkannekanti) [reply src/icons/snapchat.jsx]: not needed, removed it.

**Review by NejcZdovc — APPROVED** [review]

**Pull request merged**